tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -347,6 +347,34 @@ def test_pantsd_pid_change(self):
       # Remove the pidfile so that the teardown script doesn't try to kill process 9.
       os.unlink(pidpath)
 
+  def test_pantsd_memory_usage(self):
+    """Validates that after N runs, memory usage has increased by no more than X percent."""
+    number_of_runs = 10
+    max_memory_increase_fraction = 0.4
+    with self.pantsd_successful_run_context() as (pantsd_run, checker, workdir, config):
+      cmd = ['list', 'testprojects::']
```

Comment:
Also, from the ticket you mention:
> `(something simple and unlikely to be ported to v2 anytime soon)`

Isn't `list` already ported and more or less functional in v2?
Could it be something like `export`?

Reply:
Good call.

Reply:
Went with `filter`, which is pretty gnarly, but still only requires parsing (unlike `export`).